### DISCUSSION TITLE
Using the headObject and getObject interfaces may result in waiting timeouts.

### DISCUSSION THREAD

**Discussion by FourSpaces** [minio/minio#19886] [Q&A]
Sometimes an object in Minio may wait for a timeout, which can occur when using the headObject or getObject interfaces. We need to wait a few minutes before we can download again.
This problem is random and sometimes arises suddenly.

![image](https://github.com/minio/minio/assets/13359726/ccd386bd-5e0d-4430-8bf4-f452f61cdb28)

**Comment by klauspost** [comment]
Please use latest release.
